# Galician translation of libgda.
# Copyright (C) 2000 Jesús Bravo Álvarez.
# Jesús Bravo Álvarez <jba@pobox.com>, 2000.
#
# Proxecto Trasno - Adaptación do software libre á lingua galega:  Se desexas
# colaborar connosco, podes atopar máis información en http://trasno.gpul.org
#
# First Version: 2000-04-29 12:07+0200
#
msgid ""
msgstr ""
"Project-Id-Version: libgda\n"
"POT-Creation-Date: 2001-05-27 14:38+0200\n"
"PO-Revision-Date: 2000-10-29 15:03+0100\n"
"Last-Translator: Jesús Bravo Álvarez <jba@pobox.com>\n"
"Language-Team: Galician <trasno@ceu.fi.udc.es>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=iso-8859-1\n"
"Content-Transfer-Encoding: 8bit\n"

#: lib/gda-client/gda-connection-pool.c:337
#, c-format
msgid "could not open connection to %s"
msgstr "non foi posible abrir unha conexión a %s"

#: lib/gda-client/gda-connection-pool.c:345
#, c-format
msgid "Data source %s not found"
msgstr "Fonte de datos %s non atopada"

#: lib/gda-client/gda-connection.c:235 lib/gda-client/gda-error.c:179
#, c-format
msgid "%s: The server didn't respond."
msgstr "%s: O servidor non respondeu."

#: lib/gda-client/gda-connection.c:239 lib/gda-client/gda-error.c:183
#, c-format
msgid "%s: An Error occured in the CORBA system."
msgstr "%s: Ocorreu un erro no sistema CORBA."

#: lib/gda-client/gda-connection.c:511
#, c-format
msgid "Database '%s' not found in system configuration"
msgstr "Base de datos '%s' non atopada na configuración do sistema"

#: lib/gda-client/gda-connection.c:526
msgid "Could not open CORBA factory"
msgstr "Non foi posible abri-la fábrica CORBA"

#: lib/gda-client/gda-field.c:245
msgid "<NULL>"
msgstr "<NULL>"

#: lib/gda-client/gda-field.c:251 lib/gda-client/gda-field.c:253
msgid "<Unknown GDA Type(NULL)>"
msgstr "<Tipo GDA Descoñecido (NULL)>"

#: lib/gda-client/gda-field.c:266
msgid "TRUE"
msgstr "CERTO"

#: lib/gda-client/gda-field.c:267
msgid "FALSE"
msgstr "FALSO"

#: lib/gda-common/gda-corba.c:75
#, c-format
msgid "CORBA System exception: %s"
msgstr "Excepción de Sistema de CORBA: %s"

#: lib/gda-common/gda-corba.c:105
msgid "True"
msgstr "Certo"

#: lib/gda-common/gda-corba.c:105
msgid "False"
msgstr "Falso"

#: lib/gda-common/gda-init.c:59
msgid "Attempt to initialize an already initialized client"
msgstr "Tentouse inicializar un cliente xa inicializado"

#: lib/gda-common/gda-log.c:48
msgid "Saving log...x"
msgstr "Gardando rexistro..."

#. compose message
#: lib/gda-common/gda-log.c:69
msgid "ERROR: "
msgstr "ERRO: "

#: lib/gda-common/gda-log.c:69
msgid "MESSAGE: "
msgstr "MENSAXE: "

#: lib/gda-common/gda-thread.c:211
msgid "thread is already running"
msgstr "o fío aínda está en execución"

#: lib/gda-common/gda-xml-database.c:286
#, c-format
msgid "table %s already exists"
msgstr "a táboa %s xa existe"

#: lib/gda-common/gda-xml-database.c:450
#, c-format
msgid "%p is not a valid table"
msgstr "%p non é unha táboa válida"

#: lib/gda-server/gda-server-error.c:185
msgid "Could not get pointer to server implementation"
msgstr "Non se pode obte-lo punteiro á implementación do servidor"

#: lib/gda-server/gda-server-impl-connection.c:122
#: lib/gda-server/gda-server-impl-connection.c:131
#: lib/gda-server/gda-server-impl-connection.c:138
#: lib/gda-server/gda-server-impl-connection.c:147
#: lib/gda-server/gda-server-impl-connection.c:154
#: lib/gda-server/gda-server-impl-connection.c:163
#: lib/gda-server/gda-server-impl-connection.c:170
#: lib/gda-server/gda-server-impl-connection.c:179
#: lib/gda-server/gda-server-impl-recordset.c:126
#: lib/gda-server/gda-server-impl-recordset.c:135
#: lib/gda-server/gda-server-impl-recordset.c:142
#: lib/gda-server/gda-server-impl-recordset.c:157
#: lib/gda-server/gda-server-impl-recordset.c:166
#: lib/gda-server/gda-server-impl-recordset.c:173
#: lib/gda-server/gda-server-impl-recordset.c:182
#: lib/gda-server/gda-server-impl-recordset.c:189
#: lib/gda-server/gda-server-impl-recordset.c:198
#: lib/gda-server/gda-server-impl-recordset.c:205
#: lib/gda-server/gda-server-impl-recordset.c:213
#: lib/gda-server/gda-server-impl-recordset.c:222
#: lib/gda-server/gda-server-impl-recordset.c:229
#: lib/gda-server/gda-server-impl-recordset.c:237
#: lib/gda-server/gda-server-impl-recordset.c:245
#: lib/gda-server/gda-server-impl-recordset.c:273
#: lib/gda-server/gda-server-impl-recordset.c:281
#: lib/gda-server/gda-server-impl-recordset.c:289
#: lib/gda-server/gda-server-impl-recordset.c:297
#: lib/gda-server/gda-server-impl-recordset.c:305
#: lib/gda-server/gda-server-impl-recordset.c:314
#, c-format
msgid "%s not implemented"
msgstr "%s non implementado"

#: lib/gda-server/gda-server-impl-connection.c:269
#, c-format
msgid "impl_GDA_Connection_open: opening connection with DSN: %s"
msgstr "impl_GDA_Connection_open: abrindo a conexión co DSN: %s"

#: lib/gda-server/gda-server-impl-recordset.c:402
#, c-format
msgid "%s: an error ocurred while fetching data"
msgstr "%s: ocorreu un erro recibindo os datos"

#: lib/gda-server/gda-server-impl.c:197
#, c-format
msgid "Starting provider %s with no implementation functions"
msgstr "Iniciando o fornecedor %s sen funcións de implementación"

#: lib/gda-server/gda-server-impl.c:223
#: report/server/gda-report-server-main.c:113
msgid ""
"OAF doesn't know about our IID; indicates broken installation. Exiting..."
msgstr ""
"OAF descoñece o noso IID; isto indica unha instalación errónea. Saíndo..."

#: lib/gda-server/gda-server-impl.c:226
msgid ""
"Another instance of this provider is already registered with OAF. Exiting..."
msgstr "Xa hai rexistrada outra instancia deste fornecedor con OAF. Saíndo..."

#: lib/gda-server/gda-server-impl.c:229
msgid "Unknown error registering this provider with OAF. Exiting"
msgstr "Erro descoñecido rexistrando este fornecedor con OAF. Saíndo"

#: lib/gda-server/gda-server-impl.c:233
#, c-format
msgid "Registered with ID = %s"
msgstr "Rexistrado co ID = %s"

#: lib/gda-server/gda-server-impl.c:312
#, c-format
msgid "CORBA system exception %s"
msgstr "Excepción de sistema de CORBA %s"

#: lib/gda-server/gda-server-impl.c:315
#, c-format
msgid "CORBA user exception: %s"
msgstr "Excepción de usuario de CORBA: %s"

#: lib/gda-server/gda-server-init.c:63 lib/gda-server/gda-server-init.c:69
#, c-format
msgid "Received signal %d, shutting down"
msgstr "Recibiuse o sinal %d, saíndo"

#: lib/gda-server/gda-server-init.c:73
#, c-format
msgid "Received signal %d. Ignoring"
msgstr "Recibiuse o sinal %d. Ignorando"

#: lib/gda-server/gda-server-init.c:123
msgid "Attempt to initialize an already initialized provider"
msgstr "Tentouse inicializar un fornecedor xa inicializado"

#: providers/gda-ldap-server/GNOME_GDA_Provider_LDAP.oaf.in.h:1
msgid "GDA Datasource Access for LDAP"
msgstr ""

#: providers/gda-interbase-server/gda-interbase-connection.c:273
#: providers/gda-ldap-server/gda-ldap-connection.c:284
#: providers/gda-mdb-server/gda-mdb-connection.c:372
#: providers/gda-mysql-server/gda-mysql-connection.c:370
#: providers/gda-odbc-server/gda-odbc-connection.c:621
#: providers/gda-odbc-server/gda-odbc-connection.c:654
#: providers/gda-oracle-server/gda-oracle-connection.c:438
#: providers/gda-postgres-server/gda-postgres-connection.c:743
#: providers/gda-sybase-server/gda-sybase-error.c:115
#: providers/gda-tds-server/gda-tds-error.c:114
#, c-format
msgid "error '%s' at %s"
msgstr "erro '%s' en %s"

#: providers/gda-interbase-server/gda-interbase-connection.c:278
#: providers/gda-interbase-server/gda-interbase-connection.c:279
#: providers/gda-ldap-server/gda-ldap-connection.c:289
#: providers/gda-ldap-server/gda-ldap-connection.c:290
#: providers/gda-mdb-server/gda-mdb-connection.c:376
#: providers/gda-mdb-server/gda-mdb-connection.c:377
#: providers/gda-mysql-server/gda-mysql-connection.c:374
#: providers/gda-mysql-server/gda-mysql-connection.c:375
#: providers/gda-odbc-server/gda-odbc-connection.c:623
#: providers/gda-odbc-server/gda-odbc-connection.c:624
#: providers/gda-odbc-server/gda-odbc-connection.c:656
#: providers/gda-odbc-server/gda-odbc-connection.c:657
#: providers/gda-oracle-server/gda-oracle-connection.c:443
#: providers/gda-oracle-server/gda-oracle-connection.c:444
#: providers/gda-postgres-server/gda-postgres-connection.c:750
#: providers/gda-postgres-server/gda-postgres-connection.c:751
#: providers/gda-primebase-server/gda-primebase-connection.c:363
#: providers/gda-primebase-server/gda-primebase-connection.c:364
#: providers/gda-sybase-server/gda-sybase-error.c:124
#: providers/gda-sybase-server/gda-sybase-error.c:125
#: providers/gda-tds-server/gda-tds-error.c:123
#: providers/gda-tds-server/gda-tds-error.c:124
msgid "Not available"
msgstr "Non dispoñible"

#: providers/gda-interbase-server/gda-interbase-connection.c:280
#: providers/gda-ldap-server/gda-ldap-connection.c:291
#: providers/gda-mdb-server/gda-mdb-connection.c:378
#: providers/gda-mysql-server/gda-mysql-connection.c:376
#: providers/gda-oracle-server/gda-oracle-connection.c:445
#: providers/gda-postgres-server/gda-postgres-connection.c:752
#: providers/gda-primebase-server/gda-primebase-connection.c:365
#: providers/gda-sybase-server/gda-sybase-error.c:126
#: providers/gda-tds-server/gda-tds-error.c:125
msgid "error"
msgstr "erro"

#: providers/gda-ldap-server/main-ldap.c:71
msgid "Could not register LDAP provider implementation"
msgstr "Non foi posible rexistra-la implementación do fornecedor LDAP"

#: providers/gda-interbase-server/GNOME_GDA_Provider_InterBase.oaf.in.h:1
msgid "GDA Datasource Access for Interbase"
msgstr ""

#: providers/gda-interbase-server/gda-interbase-connection.c:272
#, c-format
msgid "Error code %ld"
msgstr "Código de erro %ld"

#: providers/gda-interbase-server/main-interbase.c:72
msgid "Could not register INTERBASE provider implementation"
msgstr "Non foi posible rexistra-la implementación do fornecedor INTERBASE"

#: providers/gda-mail-server/GNOME_GDA_Provider_Mail.oaf.in.h:1
msgid "GDA Datasource Access for Mail"
msgstr ""

#: providers/gda-mail-server/gda-mail-connection.c:108
#: providers/gda-mdb-server/gda-mdb-connection.c:224
#: providers/gda-mysql-server/gda-mysql-connection.c:239
#: providers/gda-oracle-server/gda-oracle-connection.c:295
#: providers/gda-postgres-server/gda-postgres-connection.c:477
#: providers/gda-sybase-server/gda-sybase-connection.c:283
#: providers/gda-tds-server/gda-tds-connection.c:288
#, c-format
msgid "Unhandled SCHEMA_QTYPE %d"
msgstr "SCHEMA_QTYPE %d non manexado"

#: providers/gda-mail-server/main-mail.c:72
msgid "Could not register Mail provider implementation"
msgstr "Non foi posible rexistra-la implementación do fornecedor de correo"

#: providers/gda-mdb-server/GNOME_GDA_Provider_MDB.oaf.in.h:1
msgid "GDA Datasource Access for MDB (MS Access) files"
msgstr ""

#. execute command
#: providers/gda-mdb-server/gda-mdb-command.c:86
#: providers/gda-mysql-server/gda-mysql-command.c:103
#: providers/gda-sybase-server/gda-sybase-command.c:132
#: providers/gda-tds-server/gda-tds-command.c:120
#, c-format
msgid "executing command '%s'"
msgstr "executando o comando '%s'"

#: providers/gda-mdb-server/gda-mdb-command.c:88
#: providers/gda-mysql-server/gda-mysql-command.c:105
#, c-format
msgid "info '%s'"
msgstr "info '%s'"

#: providers/gda-mdb-server/gda-mdb-connection.c:137
#: providers/gda-mysql-server/gda-mysql-connection.c:140
#, c-format
msgid ""
"%s:%d: You cannot provide a UNIX_SOCKET if you also provide either a HOST or "
"a PORT. Please remove the UNIX_SOCKET, or remove both the HOST and PORT "
"options"
msgstr ""
"%s:%d: Non pode indicar un UNIX_SOCKET se tamén dá unha MÁQUINA ou un PORTO. "
"Quite o UNIX_SOCKET, ou as opcións MÁQUINA e PORTO"

#: providers/gda-mdb-server/gda-mdb-connection.c:147
#, c-format
msgid "%s:%d: You must  provide the filename with the MS Access database"
msgstr "%s:%d: Ten que indica-lo nome do ficheiro coa base de datos MS Access"

#: providers/gda-mdb-server/gda-mdb-connection.c:152
#, c-format
msgid "Opening connection with filename=%s, user=%s, password=%s"
msgstr "Abrindo a conexión con ficheiro=%s, usuario=%s, contrasinal=%s"

#: providers/gda-mdb-server/gda-mdb-connection.c:427
#: providers/gda-mysql-server/gda-mysql-connection.c:421
msgid "mysql_recset is NULL"
msgstr "mysql_recset é NULL"

#: providers/gda-mdb-server/gda-mdb-connection.c:469
#: providers/gda-mysql-server/gda-mysql-connection.c:460
#: providers/gda-sybase-server/gda-sybase-connection.c:458
#: providers/gda-tds-server/gda-tds-connection.c:457
msgid "No table name given for SCHEMA_COLS"
msgstr "Non se indicou un nome de táboa para SCHEMA_COLS"

#: providers/gda-mdb-server/main-mdb.c:72
#: providers/gda-mysql-server/main-mysql.c:72
msgid "Could not register MySQL provider implementation"
msgstr "Non foi posible rexistra-la implementación do fornecedor MySQL"

#: providers/gda-mysql-server/GNOME_GDA_Provider_MySQL.oaf.in.h:1
msgid "GDA Datasource Access for MySQL"
msgstr ""

#: providers/gda-mysql-server/gda-mysql-connection.c:153
#, c-format
msgid ""
"Opening connection with user=%s, password=%s, host=%s, port=%s, unix_socket=%"
"s"
msgstr ""
"Abrindo a conexión con usuario=%s, contrasinal=%s, máquina=%s, porto=%s, "
"unix_socket=%s"

#: providers/gda-odbc-server/GNOME_GDA_Provider_ODBC.oaf.in.h:1
msgid "GDA Datasource Access for ODBC"
msgstr ""

#: providers/gda-odbc-server/gda-odbc-connection.c:280
#, c-format
msgid "gda_odbc_connection_open_schema: Unhandled SCHEMA_QTYPE %d\n"
msgstr "gda_odbc_connection_open_schema: SCHEMA_QTYPE %d non manexado\n"

#: providers/gda-odbc-server/gda-odbc-connection.c:638
#: providers/gda-odbc-server/gda-odbc-connection.c:639
#: providers/gda-odbc-server/gda-odbc-connection.c:640
#: providers/gda-odbc-server/gda-odbc-connection.c:671
#: providers/gda-odbc-server/gda-odbc-connection.c:672
#: providers/gda-odbc-server/gda-odbc-connection.c:673
msgid "no text"
msgstr "sen texto"

#: providers/gda-odbc-server/main-odbc.c:71
msgid "Could not register ODBC provider implementation"
msgstr "Non foi posible rexistra-la implementación do fornecedor ODBC"

#: providers/gda-oracle-server/GNOME_GDA_Provider_Oracle.oaf.in.h:1
msgid "GDA Datasource Access for Oracle"
msgstr ""

#: providers/gda-oracle-server/gda-oracle-command.c:185
msgid "Could not allocate statement handle"
msgstr "Non foi posible asignar memoria para o manexador da declaración"

#: providers/gda-oracle-server/gda-oracle-command.c:189
msgid "command object being created without associated connection"
msgstr "o obxecto de comando estase a crear sen unha conexión asociada"

#: providers/gda-oracle-server/gda-oracle-command.c:220
#, c-format
msgid "Executing command '%s'"
msgstr "Executando o comando '%s'"

#: providers/gda-oracle-server/gda-oracle-connection.c:113
msgid "error in OCIHandleAlloc() HTYPE_SVCCTX"
msgstr "erro en OCIHandleAlloc() HTYPE_SVCCTX"

#: providers/gda-oracle-server/gda-oracle-connection.c:116
msgid "error in OCIHandleAlloc() HTYPE_SERVER"
msgstr "erro en OCIHandleAlloc() HTYPE_SERVER"

#: providers/gda-oracle-server/gda-oracle-connection.c:119
msgid "error in OCIHandleAlloc() HTYPE_ERROR"
msgstr "erro en OCIHandleAlloc() HTYPE_ERROR"

#: providers/gda-oracle-server/gda-oracle-connection.c:379
#, c-format
msgid "Unknown ORACLE type %ld"
msgstr "Tipo %ld de ORACLE descoñecido"

#: providers/gda-oracle-server/gda-oracle-recordset.c:115
msgid "Unknown ORACLE data type in fill_field_values"
msgstr "Tipo de datos de ORACLE descoñecido en fill_field_values"

#: providers/gda-oracle-server/gda-oracle-recordset.c:150
msgid "recordset object being created without associated connection"
msgstr "o obxecto recordset estase a crear sen ningunha conexión asociada"

#: providers/gda-oracle-server/main-oracle.c:71
msgid "Could not register ORACLE provider implementation"
msgstr "Non foi posible rexistra-la implementación do fornecedor ORACLE"

#: providers/gda-postgres-server/GNOME_GDA_Provider_Postgres.oaf.in.h:1
msgid "GDA Datasource Access for Postgres"
msgstr ""

#: providers/gda-postgres-server/gda-postgres-connection.c:849
#: providers/gda-postgres-server/gda-postgres-connection.c:957
msgid "postgres_recset is NULL"
msgstr "postgres_recset é NULL"

#: providers/gda-postgres-server/main-postgres.c:71
msgid "Could not register PostgreSQL provider implementation"
msgstr "Non foi posible rexistra-la implementación do fornecedor PostgreSQL"

#: providers/gda-sybase-server/GNOME_GDA_Provider_Sybase.oaf.in.h:1
msgid "GDA Datasource Access for Sybase"
msgstr ""

#.      gda_server_error_make(error, 0, cnc, "connection seems to be dead");
#: providers/gda-sybase-server/gda-sybase-command.c:127
#: providers/gda-tds-server/gda-tds-command.c:116
msgid "Connection seems to be dead"
msgstr "Semella que a conexión está morta"

#: providers/gda-sybase-server/gda-sybase-command.c:146
#: providers/gda-tds-server/gda-tds-command.c:131
msgid "Filling command structure failed\n"
msgstr "Fallou o enchido da estructura do comando\n"

#: providers/gda-sybase-server/gda-sybase-command.c:158
#: providers/gda-tds-server/gda-tds-command.c:139
msgid "sending cmd failed"
msgstr "fallou o envío do comando"

#: providers/gda-sybase-server/gda-sybase-command.c:165
#: providers/gda-tds-server/gda-tds-command.c:146
msgid "Allocating recordset failed"
msgstr "Fallou a asignación de memoria para o recordset"

#: providers/gda-sybase-server/gda-sybase-command.c:175
#: providers/gda-tds-server/gda-tds-command.c:154
msgid "Getting recordset userdata failed"
msgstr "Fallou a obtención dos datos de usuario do recordset"

#: providers/gda-sybase-server/gda-sybase-command.c:233
#: providers/gda-tds-server/gda-tds-command.c:207
msgid "Command execution finished, all results processed"
msgstr "Execución do comando rematada, procesados tódolos resultados"

#: providers/gda-sybase-server/gda-sybase-command.c:237
#: providers/gda-tds-server/gda-tds-command.c:211
msgid "Command failed"
msgstr "Comando fallido"

#: providers/gda-sybase-server/gda-sybase-command.c:241
#: providers/gda-tds-server/gda-tds-command.c:215
msgid "Execution of resultless command succeeded"
msgstr "Execución correcta dun comando sen resultado"

#: providers/gda-sybase-server/gda-sybase-command.c:257
#: providers/gda-tds-server/gda-tds-command.c:231
#, c-format
msgid "ct_results() returned %x"
msgstr "ct_results() devolveu %x"

#: providers/gda-sybase-server/gda-sybase-command.c:267
#: providers/gda-tds-server/gda-tds-command.c:241
msgid "No further results, command completly succeeded."
msgstr "Non hai máis resultados, o comando completouse correctamente."

#: providers/gda-sybase-server/gda-sybase-command.c:271
#: providers/gda-tds-server/gda-tds-command.c:245
msgid "An error occured during command execution."
msgstr "Ocorreu un erro durante a execución do comando."

#: providers/gda-sybase-server/gda-sybase-command.c:278
#: providers/gda-tds-server/gda-tds-command.c:249
msgid "Cmd execution returned unknown result, may be an error"
msgstr ""
"A execución do comando devolveu un resultado descoñecido, pode que sexa un "
"erro"

#: providers/gda-sybase-server/gda-sybase-command.c:291
#: providers/gda-tds-server/gda-tds-command.c:259
msgid "No data returned, freeing recordset structure"
msgstr "Non se devolveu ningún dato, liberando a estructura recordset"

#: providers/gda-sybase-server/gda-sybase-command.c:322
#: providers/gda-sybase-server/gda-sybase-command.c:334
#: providers/gda-tds-server/gda-tds-command.c:289
#: providers/gda-tds-server/gda-tds-command.c:297
msgid "Freeing command structure failed"
msgstr "Fallou a liberación da estructura do comando"

#: providers/gda-sybase-server/gda-sybase-connection.c:171
msgid "Not enough memory for sybase_Connection"
msgstr "Non hai memoria dabondo para sybase_Connection"

#: providers/gda-sybase-server/gda-sybase-connection.c:448
#: providers/gda-tds-server/gda-tds-connection.c:447
#, c-format
msgid "schema_cols: table_name '%s'"
msgstr "schema_cols: table_name '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:464
#: providers/gda-tds-server/gda-tds-connection.c:463
msgid "Could not allocate command structure"
msgstr "Non foi posible asignar memoria para a estructura de comando"

#: providers/gda-sybase-server/gda-sybase-connection.c:486
#: providers/gda-tds-server/gda-tds-connection.c:485
msgid "Could not allocate enough memory for query"
msgstr "Non foi posible asignar memoria dabondo para a consulta"

#. 		gda_server_error_make(error, NULL, cnc, __PRETTY_FUNCTION__);
#: providers/gda-sybase-server/gda-sybase-connection.c:500
#: providers/gda-tds-server/gda-tds-connection.c:499
msgid "Could not get recordset in schema_cols"
msgstr "Non foi posible obte-lo recordset en schema_cols"

#: providers/gda-sybase-server/gda-sybase-connection.c:549
#: providers/gda-tds-server/gda-tds-connection.c:548
#, c-format
msgid "schema_procs: invalid constraint type %d"
msgstr "schema_procs: tipo de restricción %d non válida"

#: providers/gda-sybase-server/gda-sybase-connection.c:559
#: providers/gda-tds-server/gda-tds-connection.c:558
msgid "Could not allocate cmd structure"
msgstr "Non foi posible asignar memoria para a estructura cmd"

#: providers/gda-sybase-server/gda-sybase-connection.c:626
#: providers/gda-tds-server/gda-tds-connection.c:625
#, c-format
msgid "schema_tables: invalid constraint type %d"
msgstr "schema_tables: tipo de restricción %d non válida"

#: providers/gda-sybase-server/gda-sybase-connection.c:687
#: providers/gda-tds-server/gda-tds-connection.c:686
#, c-format
msgid "Invalid or unsupported constraint type: %d"
msgstr "Tipo de restricción non válida ou non soportada: %d"

#: providers/gda-sybase-server/gda-sybase-connection.c:728
#: providers/gda-tds-server/gda-tds-connection.c:727
msgid "Could not allocate GdaServerCommand"
msgstr "Non foi posible asignar memoria para GdaServerCommand"

#: providers/gda-sybase-server/gda-sybase-connection.c:758
#: providers/gda-sybase-server/gda-sybase-connection.c:763
#: providers/gda-tds-server/gda-tds-connection.c:757
#: providers/gda-tds-server/gda-tds-connection.c:762
#, c-format
msgid "connection status: %x"
msgstr "estado da conexión: %x"

#: providers/gda-sybase-server/gda-sybase-connection.c:759
#: providers/gda-tds-server/gda-tds-connection.c:758
#, c-format
msgid "return code: %x"
msgstr "código devolvido: %x"

#: providers/gda-sybase-server/gda-sybase-connection.c:796
#: providers/gda-tds-server/gda-tds-connection.c:793
msgid "attempting reconnection"
msgstr "tentando a reconexión"

#: providers/gda-sybase-server/gda-sybase-connection.c:935
#, c-format
msgid "Reconnected to '%s'"
msgstr "Reonectado a '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:937
#: providers/gda-tds-server/gda-tds-connection.c:898
#, c-format
msgid "Connected to %s"
msgstr "Conectado a %s"

#: providers/gda-sybase-server/gda-sybase-connection.c:945
#: providers/gda-tds-server/gda-tds-connection.c:908
#, c-format
msgid "Selected database '%s'"
msgstr "Seleccionada a base de datos '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:974
#: providers/gda-tds-server/gda-tds-connection.c:933
#, c-format
msgid "%s: Could not allocate cmd structure"
msgstr "%s: Non foi posible asignar memoria para a estructura do comando"

#: providers/gda-sybase-server/gda-sybase-connection.c:983
#: providers/gda-tds-server/gda-tds-connection.c:942
#, c-format
msgid "%s: Could not allocate memory for cmd string"
msgstr "%s: Non foi posible asignar memoria para a cadea do comando"

#: providers/gda-sybase-server/gda-sybase-connection.c:996
#: providers/gda-tds-server/gda-tds-connection.c:952
#, c-format
msgid "%s: ct_command failed"
msgstr "%s: ct_command fallou"

#: providers/gda-sybase-server/gda-sybase-connection.c:1005
#: providers/gda-tds-server/gda-tds-connection.c:961
#, c-format
msgid "%s: sending command failed"
msgstr "%s: fallou o envío do comando"

#: providers/gda-sybase-server/gda-sybase-connection.c:1016
#, c-format
msgid "%s: selecting database failed"
msgstr "%s: fallou a selección da base de datos"

#: providers/gda-sybase-server/gda-sybase-connection.c:1034
#, c-format
msgid "%s: use command failed"
msgstr "%s: o comando 'use' fallou"

#: providers/gda-sybase-server/gda-sybase-connection.c:1061
#: providers/gda-tds-server/gda-tds-connection.c:1023
msgid "Could not allocate locale structure"
msgstr "Non foi posible asignar memoria para a estructura de locale"

#: providers/gda-sybase-server/gda-sybase-connection.c:1067
#: providers/gda-tds-server/gda-tds-connection.c:1029
#, c-format
msgid "Could not set locale to '%s'"
msgstr "Non foi posible establece-lo locale a '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:1073
#: providers/gda-tds-server/gda-tds-connection.c:1035
#, c-format
msgid "Could not configure locale to be '%s'"
msgstr "Non foi posible configura-lo locale como '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:1078
#: providers/gda-tds-server/gda-tds-connection.c:1040
msgid "Could not drop locale structure"
msgstr "Non foi posible libera-la memoria da estructura de locale"

#: providers/gda-sybase-server/gda-sybase-connection.c:1091
#: providers/gda-tds-server/gda-tds-connection.c:1053
#, c-format
msgid "Fatal error: %s\n"
msgstr "Erro moi grave: %s\n"

#: providers/gda-sybase-server/gda-sybase-error.c:71
#, c-format
msgid "gda_sybase_error_make(%p,%p,%p,%p)"
msgstr "gda_sybase_error_make(%p,%p,%p,%p)"

#: providers/gda-sybase-server/gda-sybase-error.c:80
#: providers/gda-tds-server/gda-tds-error.c:78
#, c-format
msgid "No error flag ist set, though %s is called. This may be a bug."
msgstr ""
"Non hai ningunha marca de erro activa, pero chamouse a %s. Isto pode ser un "
"erro no código."

#: providers/gda-sybase-server/gda-sybase-error.c:144
msgid "cs_diag() initialized"
msgstr "cs_diag() inicializado"

#: providers/gda-sybase-server/gda-sybase-error.c:147
msgid "Could not initialize cs_diag()"
msgstr "Non foi posible inicializar cs_diag()"

#: providers/gda-sybase-server/gda-sybase-error.c:154
msgid "ct_diag() initialized"
msgstr "ct_diag() inicializado"

#: providers/gda-sybase-server/gda-sybase-error.c:157
msgid "Could not initialize ct_diag()"
msgstr "Non foi posible inicializar ct_diag()"

#: providers/gda-sybase-server/gda-sybase-error.c:191
#, c-format
msgid "%s could not find server implementation"
msgstr "%s non puido atopa-la implementación do servidor"

#: providers/gda-sybase-server/gda-sybase-error.c:197
#, c-format
msgid "%s could not get cnc userdata"
msgstr "%s non puido obte-los datos de usuario de cnc"

#: providers/gda-sybase-server/gda-sybase-error.c:209
msgid "Could not get # of messages"
msgstr "Non foi posible obte-lo número de mensaxes"

#: providers/gda-sybase-server/gda-sybase-error.c:214
#, c-format
msgid "%d ctlib messages to fetch"
msgstr "%d mensaxes de ctlib para recibir"

#: providers/gda-sybase-server/gda-sybase-error.c:216
#, c-format
msgid "%d ctlib message to fetch"
msgstr "%d mensaxes de ctlib para recibir"

#: providers/gda-sybase-server/gda-sybase-error.c:252
msgid "error is nullpointer in gda_sybase_messages_handle"
msgstr "o erro é un punteiro a null en gda_sybase_messages_handle"

#: providers/gda-sybase-server/gda-sybase-error.c:277
#: providers/gda-sybase-server/gda-sybase-error.c:344
#: providers/gda-sybase-server/gda-sybase-error.c:387
msgid "Could not get # of server messages"
msgstr "Non foi posible obte-lo número de mensaxes de servidor"

#: providers/gda-sybase-server/gda-sybase-error.c:285
#, c-format
msgid "Fetching %d client messages"
msgstr "Recibindo %d mensaxes de cliente"

#: providers/gda-sybase-server/gda-sybase-error.c:287
#, c-format
msgid "Fetching %d client message"
msgstr "Recibindo %d mensaxe de cliente"

#: providers/gda-sybase-server/gda-sybase-error.c:296
#, c-format
msgid "Could not get client msg # %d"
msgstr "Non foi posible obte-la mensaxe de cliente %d"

#: providers/gda-sybase-server/gda-sybase-error.c:312
msgid "Could not get # of client messages"
msgstr "Non foi posible obte-lo número de mensaxes de cliente"

#: providers/gda-sybase-server/gda-sybase-error.c:320
msgid "Could not clear client messages"
msgstr "Non foi posible baleira-las mensaxes de cliente"

#: providers/gda-sybase-server/gda-sybase-error.c:352
#, c-format
msgid "Fetching %d server messages"
msgstr "Recibindo %d mensaxes de servidor"

#: providers/gda-sybase-server/gda-sybase-error.c:354
#, c-format
msgid "Fetching %d server message"
msgstr "Recibindo %d mensaxe de servidor"

#: providers/gda-sybase-server/gda-sybase-error.c:363
#, c-format
msgid "Could not get server msg # %d"
msgstr "Non foi posible recibi-la mensaxe de servidor %d"

#: providers/gda-sybase-server/gda-sybase-error.c:395
msgid "Could not clear server messages"
msgstr "Non foi posible baleira-las mensaxes de servidor"

#: providers/gda-sybase-server/gda-sybase-error.c:417
#: providers/gda-sybase-server/gda-sybase-error.c:453
msgid "Could not get # of cslib messages"
msgstr "Non foi posible obte-lo número de mensaxes de cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:425
#, c-format
msgid "Fetching %d cslib messages"
msgstr "Recibindo %d mensaxes de cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:427
#, fuzzy, c-format
msgid "Fetching %d cslib message"
msgstr "Recibindo %d mensaxes de cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:437
#, c-format
msgid "Could not get cslib msg # %d"
msgstr "Non foi posible recibi-la mensaxe de cslib %d"

#: providers/gda-sybase-server/gda-sybase-error.c:461
msgid "Could not clear cslib messages"
msgstr "Non foi posible baleira-las mensaxes de cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:593
#: providers/gda-tds-server/gda-tds-error.c:257
msgid "Server message:"
msgstr "Mensaxe de servidor:"

#: providers/gda-sybase-server/gda-sybase-error.c:602
#: providers/gda-tds-server/gda-tds-error.c:266
msgid "Client message:"
msgstr "Mensaxe de cliente:"

#: providers/gda-sybase-server/gda-sybase-error.c:609
#: providers/gda-tds-server/gda-tds-error.c:273
msgid "CS-Library error:"
msgstr "Erro de CS-Library:"

#: providers/gda-sybase-server/gda-sybase-recordset.c:158
#: providers/gda-tds-server/gda-tds-recordset.c:155
#, c-format
msgid "fetch error at row %d"
msgstr "erro recibindo na ringleira %d"

#: providers/gda-sybase-server/gda-sybase-recordset.c:165
#: providers/gda-tds-server/gda-tds-recordset.c:162
#, c-format
msgid "CS_END_DATA on row %d; canceling"
msgstr "CS_END_DATA na ringleira %d; cancelando"

#: providers/gda-sybase-server/gda-sybase-recordset.c:174
#: providers/gda-tds-server/gda-tds-recordset.c:169
#, c-format
msgid "%s: Error (%d)"
msgstr "%s: Erro (%d)"

#: providers/gda-sybase-server/gda-sybase-recordset.c:349
#: providers/gda-tds-server/gda-tds-recordset.c:342
msgid "Failed fetching # of columns"
msgstr "Fallou a recepción do número de columnas"

#: providers/gda-sybase-server/gda-sybase-recordset.c:355
#: providers/gda-tds-server/gda-tds-recordset.c:349
#, c-format
msgid "Counted %d columns"
msgstr "Contáronse %d columnas"

#: providers/gda-sybase-server/gda-sybase-recordset.c:362
#: providers/gda-tds-server/gda-tds-recordset.c:356
#, c-format
msgid "%s could not allocate datafmt info"
msgstr "%s non puido asignar memoria para a información do formato de datos"

#: providers/gda-sybase-server/gda-sybase-recordset.c:372
#: providers/gda-tds-server/gda-tds-recordset.c:366
#, c-format
msgid "%s could not allocate data fields"
msgstr "%s non puido asignar memoria para os campos de datos"

#: providers/gda-sybase-server/gda-sybase-recordset.c:390
#: providers/gda-tds-server/gda-tds-recordset.c:384
#, c-format
msgid "ct_describe on col # %d"
msgstr "ct_describe na columna %d"

#: providers/gda-sybase-server/gda-sybase-recordset.c:397
#: providers/gda-tds-server/gda-tds-recordset.c:391
#, c-format
msgid "ct_describe on col # %d failed"
msgstr "ct_describe na columna %d fallou"

#: providers/gda-sybase-server/gda-sybase-recordset.c:403
#: providers/gda-tds-server/gda-tds-recordset.c:397
#, c-format
msgid "ct_describe(%d): '%s', %d (%s)"
msgstr "ct_describe(%d): '%s', %d (%s)"

#: providers/gda-sybase-server/gda-sybase-recordset.c:412
#: providers/gda-tds-server/gda-tds-recordset.c:406
msgid "could not allocate data holder"
msgstr "non foi posible asignar memoria para o almacenador de datos"

#: providers/gda-sybase-server/gda-sybase-recordset.c:422
#: providers/gda-tds-server/gda-tds-recordset.c:416
msgid "could not ct_bind data holder to column"
msgstr "non foi posible asignar con ct_bind o almacenador de datos á columna"

#: providers/gda-sybase-server/gda-sybase-recordset.c:429
#: providers/gda-tds-server/gda-tds-recordset.c:423
msgid "illegal type detected, aborting"
msgstr "tipo ilegal detectado, abortando"

#: providers/gda-sybase-server/gda-sybase-recordset.c:435
#: providers/gda-tds-server/gda-tds-recordset.c:429
msgid "could not allocate field"
msgstr "non foi posible asignar memoria para o campo"

#: providers/gda-sybase-server/main-sybase.c:76
msgid "Could not register gda-sybase provider implementation"
msgstr "Non foi posible rexistra-la implementación do fornecedor gda-sybase"

#: providers/gda-primebase-server/GNOME_GDA_Provider_Primebase.oaf.in.h:1
msgid "GDA Datasource Access for primebase"
msgstr ""

#: providers/gda-primebase-server/gda-primebase-connection.c:158
#, fuzzy, c-format
msgid "Could not connect to '%s@%s:%s' with connparm '%s'"
msgstr "non foi posible abrir unha conexión a %s"

#: providers/gda-primebase-server/gda-primebase-connection.c:162
#, c-format
msgid "%2ld: Connected to '%s@%s:%s' with connparm: '%s'"
msgstr ""

#: providers/gda-primebase-server/gda-primebase-connection.c:168
msgid "Could not get PBIConnectionInfo(), disconnecting"
msgstr ""

#: providers/gda-primebase-server/gda-primebase-connection.c:172
#, c-format
msgid "%2ld: ConInfo: '%s@%s', Cl v%x, Srv '%s' v%x, Brand '%s'"
msgstr ""

#: providers/gda-primebase-server/gda-primebase-connection.c:376
#, fuzzy
msgid "Not enough memory for primebase_Error structure."
msgstr "Non hai memoria dabondo para sybase_Connection"

#: providers/gda-primebase-server/gda-primebase-connection.c:383
#, c-format
msgid "%2ld: ERROR: %ld (%ld) : %s.\n"
msgstr ""

#: providers/gda-primebase-server/main-primebase.c:70
#, fuzzy
msgid "Could not register PRIMEBASE provider implementation"
msgstr "Non foi posible rexistra-la implementación do fornecedor INTERBASE"

#: providers/gda-tds-server/GNOME_GDA_Provider_TDS.oaf.in.h:1
msgid "GDA Datasource Access for Sybase/MS SQL"
msgstr ""

#: providers/gda-tds-server/gda-tds-connection.c:167
#, fuzzy
msgid "Not enough memory for tds_Connection"
msgstr "Non hai memoria dabondo para sybase_Connection"

#. Fixme: replace "dummy" with buf when CS_GET CS_SERVERNAME works
#: providers/gda-tds-server/gda-tds-connection.c:896
#, fuzzy, c-format
msgid "Reconnected to %s"
msgstr "Reonectado a '%s'"

#: providers/gda-tds-server/gda-tds-error.c:69
#, fuzzy, c-format
msgid "gda_tds_error_make(%p,%p,%p,%p)"
msgstr "gda_sybase_error_make(%p,%p,%p,%p)"

#: providers/gda-tds-server/main-tds.c:76
#, fuzzy
msgid "Could not register gda-tds provider implementation"
msgstr "Non foi posible rexistra-la implementación do fornecedor gda-sybase"

#: report/gda-report-engine.c:209
#, c-format
msgid "CORBA exception unloading report engine: %s"
msgstr "Excepción de CORBA o descargar o motor de informes: %s"

#: report/server/GNOME_GDA_Report.oaf.in.h:1
msgid "GDA Distributed Report Engine"
msgstr ""

#: report/server/gda-report-server-main.c:41
#: report/server/gda-report-server-main.c:48
#, c-format
msgid "Received signal %d, shutting down."
msgstr "Recibiuse o sinal %d, saíndo."

#: report/server/gda-report-server-main.c:52
#, c-format
msgid "Received signal %d, shutting down cleanly"
msgstr "Recibiuse o sinal %d, saíndo limpamente"

#: report/server/gda-report-server-main.c:92
msgid "Failed to initialize OAF: please mail bug report to OAF maintainers"
msgstr ""
"Fallo ó inicializar OAF: por favor, envíe un informe do erro ós encargados "
"do OAF"

#: report/server/gda-report-server-main.c:101
msgid "Could not activate report engine"
msgstr "Non foi posible activa-lo motor de informes"

#: report/server/gda-report-server-main.c:116
msgid ""
"Another instance of the report engine is already registered with OAF. "
"Exiting..."
msgstr ""
"Xa hai rexistrada outra instancia do motor de informes con OAF. Saíndo..."

#: report/server/gda-report-server-main.c:119
msgid "Unknown error registering report engine with OAF. Exiting"
msgstr "Erro descoñecido rexistrando o motor de informes con OAF. Saíndo"

#: report/server/gda-report-server-main.c:123
#, c-format
msgid "Registered report engine with ID = %s"
msgstr "Rexistrado o motor de informes co ID = %s"

#: tools/gda-run.c:109
#, c-format
msgid "%s: misconfigured DSN entry"
msgstr "%s: entrada DSN mal configurada"

#: tools/gda-run.c:124
#, c-format
msgid "%s: data source '%s' not found\n"
msgstr "%s: fonte de datos '%s' non atopada\n"

#~ msgid "%s: length = %d"
#~ msgstr "%s: lonxitude = %d"

#~ msgid "%s: sending field description for '%s'"
#~ msgstr "%s: enviando a descrición do campo '%s'"

#~ msgid "Error allocating error structure"
#~ msgstr "Erro asignando memoria para a estructura de erro"

#~ msgid "schema_tables: table name = '%s'"
#~ msgstr "schema_tables: table name = '%s'"

#~ msgid "schema_tables: table_schema = '%s'"
#~ msgstr "schema_tables: table_schema = '%s'"

#~ msgid "schema_procedures: proc_catalog = '%s' UNUSED!\n"
#~ msgstr "schema_procedures: proc_catalog = '%s' ¡SEN USAR!\n"

#~ msgid "schema_procedures: proc_catalog = '%s' UNUSED!"
#~ msgstr "schema_procedures: proc_catalog = '%s' ¡SEN USAR!"

#~ msgid "database name"
#~ msgstr "nome da base de datos"

#~ msgid "DATABASE"
#~ msgstr "BASE-DE-DATOS"

#~ msgid "user name"
#~ msgstr "nome de usuario"

#~ msgid "USER-NAME"
#~ msgstr "NOME-DE-USUARIO"

#~ msgid "password"
#~ msgstr "contrasinal"

#~ msgid "PASSWORD"
#~ msgstr "CONTRASINAL"

#~ msgid "file name"
#~ msgstr "nome do ficheiro"

#~ msgid "FILE-NAME"
#~ msgstr "NOME-DO-FICHEIRO"

#~ msgid "Fetchong %d cslib message"
#~ msgstr "Recibindo %d mensaxe de cslib"

#~ msgid "%s: not implemented"
#~ msgstr "%s: non implementado"
